lib/evmone/execution_state.hpp

```diff
@@ -101,11 +107,23 @@ struct ExecutionState
     evmc::HostContext host;
     evmc_revision rev = {};
     bytes return_data;
+
+    /// Reference to original EVM code.
+    /// TODO: Code should be accessed via code analysis only and this should be removed.
```

Comment:
Note: I use it in a somewhat hacky way in EOF1 implementation in Baseline to access the original (non-padded) code for `CODECOPY` / `CODESIZE`.
Don't know yet what would be a better approach (maybe keeping a reference to original code in analysis, too)

Reply:
Yes, the original code will be available in the analysis.